test/unit/test_assembly.py

```diff
@@ -312,6 +312,18 @@ def test_multi_overlap(self):
             str(Bio.SeqIO.read(outFasta, 'fasta').seq),
             str(Bio.SeqIO.read(expected, 'fasta').seq))
 
+class TestGap2Seq(TestCaseWithTmp):
+    '''Test gap-filling tool Gap2Seq'''
+
+    def test_gapfill(self):
+        from os.path import join
+        inDir = util.file.get_test_input_path()
+        in_scaffold = join(inDir, 'TestOrderAndOrient', 'expected.ebov.doublehit.fasta')
+        with util.file.tempfname(suffix='.filled.fasta') as filled:
+            assembly.gapfill_gap2seq(in_scaffold=in_scaffold,
+                                     inBam=join(inDir, 'G5012.3.testreads.bam'), out_scaffold=filled)
+            self.assertEqualContents(filled, join(inDir, 'TestGap2Seq', 'expected.ebov.doublehit.gapfill.fasta'))
```

Comment:
If the output is always the same, maybe we should check the hash rather than store the expected output verbatim. We have [`assert_md5_equal_to_line_in_file`](https://github.com/broadinstitute/viral-ngs/blob/master/test/__init__.py#L55) to help. Simon and I had some brief chatter about this on Slack in relation to his effort toward repo thinning.